src/sentry/utils/snuba.py

```diff
@@ -644,6 +644,8 @@ def raw_query(
         is_grouprelease=is_grouprelease,
         **kwargs,
     )
+    if referrer:
+        sentry_sdk.set_tag("query.referrer", referrer)
```

Comment:
Should this be in `bulk_raw_query` since alerts can directly call `bulk_raw_query` with the referrer?

Reply:
:+1: Yeah that makes more sense